living_documentation_generator/generator.py

```diff
@@ -140,73 +209,108 @@ def _fetch_github_project_issues(self) -> dict[str, ProjectIssue]:
             for project in projects:
                 logger.info("Fetching GitHub project data - from `%s`.", project.title)
-                project_issues: list[ProjectIssue] = (self.safe_call(self.github_projects_instance.get_project_issues)
-                                                      (project=project))
+                project_issues: list[ProjectIssue] = self.__safe_call(
+                    self.__github_projects_instance.get_project_issues)(
+                    project=project,
+                )
 
                 for project_issue in project_issues:
-                    key = make_issue_key(project_issue.organization_name, project_issue.repository_name,
-                                         project_issue.number)
+                    key = make_issue_key(
+                        project_issue.organization_name,
+                        project_issue.repository_name,
+                        project_issue.number,
+                    )
                     all_project_issues[key] = project_issue
-                logger.info("Fetching GitHub project data - fetched project data (%s).", project.title)
+                logger.info(
+                    "Fetching GitHub project data - fetched project data (%s).",
+                    project.title,
+                )
 
         return all_project_issues
 
     @staticmethod
-    def _consolidate_issues_data(repository_issues: dict[str, list[Issue]],
-                                 projects_issues: dict[str, ProjectIssue]) -> dict[str, ConsolidatedIssue]:
+    def _consolidate_issues_data(
+        repository_issues: dict[str, list[Issue]],
+        projects_issues: dict[str, ProjectIssue],
+    ) -> dict[str, ConsolidatedIssue]:
+        """
+        Consolidate the fetched issues and extra project data into a one consolidated object.
+
+        Args:
+            repository_issues (dict): A dictionary containing repository issue objects with unique key.
+            projects_issues (dict): A dictionary containing project issue objects with unique key.
+        """
 
         consolidated_issues = {}
 
         # Create a ConsolidatedIssue object for each repository issue
         for repository_id in repository_issues.keys():
             for repository_issue in repository_issues[repository_id]:
                 repo_id_parts = repository_id.split("/")
-                unique_key = make_issue_key(repo_id_parts[0], repo_id_parts[1], repository_issue.number)
-                consolidated_issues[unique_key] = ConsolidatedIssue(repository_id=repository_id,
-                                                                    repository_issue=repository_issue)
+                unique_key = make_issue_key(
+                    repo_id_parts[0], repo_id_parts[1], repository_issue.number
+                )
+                consolidated_issues[unique_key] = ConsolidatedIssue(
+                    repository_id=repository_id, repository_issue=repository_issue
```

Comment:
Same here.

Reply:
"Trailing comma signals to Black that you might want to add more arguments later. But since the arguments fit on a single line, Black prefers to remove the trailing comma and keep them on one line."

This pattern is consistent in the whole project.